generators/entity/templates/client/angular/src/main/webapp/app/entities/_entity.service.ts

```diff
@@ -18,7 +18,7 @@
 -%>
 <%_
     let hasDate = false;
-    if (fieldsContainZonedDateTime || fieldsContainLocalDate) {
+    if (fieldsContainInstant || fieldsContainZonedDateTime) {
```

Comment:
I think we should keep `fieldsContainLocalDate` in the condition too
I will made a PR for that